tests/test_plot_losses.py

```diff
@@ -6,15 +6,20 @@
 class CheckOutput:
     def send(self, logger: MainLogger):
         assert isinstance(logger, MainLogger)
+        grouped_log_history = logger.grouped_log_history(raw_names=True, raw_group_names=True)
+        assert len(grouped_log_history) == 2
+        assert len(grouped_log_history['accuracy']) == 2
+        assert len(grouped_log_history['accuracy']['val_acc']) == 2
         grouped_log_history = logger.grouped_log_history()
         assert len(grouped_log_history) == 2
-        assert len(grouped_log_history['acccuracy']) == 2
-        assert len(grouped_log_history['acccuracy']['val_acc']) == 2
+        assert len(grouped_log_history['accuracy']) == 2
+        print(grouped_log_history)
+        assert len(grouped_log_history['accuracy']['validation ']) == 2
 
 
 def test_plot_losses():
     """Test basic usage"""
-    group_patterns = {'acccuracy': re.compile(r'.*acc$'), 'log-loss': re.compile(r'.*loss$')}
+    group_patterns = {'accuracy': re.compile(r'.*acc$'), 'log-loss': re.compile(r'.*loss$')}
```

Comment:
Do we want to use these patterns? Do they compete with the new ones?

Reply:
In the past I thought, that they are more flexible, but currently they are not.

Reply:
removed